src/main/java/com/jme3/math/Quaternion.java

```diff
@@ -1498,6 +1502,14 @@ public void negate() {
 		w *= -1;
 	}
 
+	/**
+	 * <code>negate</code> returns the negated quaternion.
+	 *
+	 */
+	public Quaternion negated() {
```

Comment:
```suggestion
	public Quaternion negate() {
```

I know this would conflict with current `negate()` method, but it should be renamed to `negateLocal()` and return `this`. There are naming conventions for this lib, let's follow them.